test_scripts/API/SetAudioStreamingIndicator/ideas_SetAudioStreamingIndicator_conditions_for_SDL_to_handle_related_cases.txt

```diff
@@ -0,0 +1,111 @@
+./test_scripts/API/SetAudioStreamingIndicator/061_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_navi_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/062_ATF_N_TC_SetAudioStreamingIndicator_USER_DISALLOWED_comm_app_RPC_disallowed_by_user_ExternalProprietary.lua Check_that_SDL_responds_with_USER_DISALLOWED_when_RPC_disallowed_by_user_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/063_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_media_app
+./test_scripts/API/SetAudioStreamingIndicator/064_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_non-media_app
+./test_scripts/API/SetAudioStreamingIndicator/065_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_navi_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_navi_app
+./test_scripts/API/SetAudioStreamingIndicator/066_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_comm_app_RPC_omitted_PTgroups.lua Check_that_SDL_responds_with_DISALLOWED_when_RPC_omitted_PT_groups_comm_app
+./test_scripts/API/SetAudioStreamingIndicator/067_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/068_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_LIMITED_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/069_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/070_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_media_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/071_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_FULL_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_non-media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/072_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_BACKGROUND_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_non-media_app_notincluded_PT
+./test_scripts/API/SetAudioStreamingIndicator/073_ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_NONE_RPC_HMI_level_not_included.lua Check_that_SDL_responds_with_DISALLOWED_when_HMIlevel_media_non-media_notincluded_PT
```

Comment:
Test case for 'LIMITED' HMI level is missing
Please add it (ex. ATF_N_TC_SetAudioStreamingIndicator_DISALLOWED_non-media_app_LIMITED_RPC_HMI_level_not_included.lua)